Source/FileSDK/Public/FileSDKBPLibrary.h

```diff
@@ -559,3 +561,34 @@ class UFileSDKBPLibrary : public UBlueprintFunctionLibrary {
 
   static std::ios_base::seekdir FileAnchorToSeekDir(EFileSDKFileAnchor Anchor);
 };
+  /**
+   * Stores a string in your system's clipboard.
+   *
+   * @param InString String to copy.
+   */
+  
+  UFUNCTION(BlueprintCallable, 
+  meta = (
+		DisplayName="Set System Clipboard",
+		Keywords = "Copy string to clipboard"
+	),  
+  Category = "FileSDK | Clipboard"
+  )
+  static void CopyToClipboard(const FString& InString);
+  
+    /**
+   * Returns a string from your system's clipboard.
+   *
+   * @param Return String stored in your system's clipboard.
+   */
+	
+  UFUNCTION(BlueprintPure, 
+  meta = (
+		DisplayName="Get System Clipboard",
+		Keywords = "Paste string from clipboard"
+	),
+  Category = "FileSDK | Clipboard"
+  )	
+  static void PasteClipboard(FString& Return); 
+   
+};
```

Comment:
This is still just a bunch of whitespace/style changes

```suggestion

  /**
   * Stores a string in your system's clipboard.
   *
   * @param InString String to copy.
   */
  UFUNCTION(
    BlueprintCallable,
    meta = (
      DisplayName="Set System Clipboard",
      Keywords = "Copy string to clipboard"
    ),
    Category = "FileSDK | Clipboard"
  )
  static void CopyToClipboard(const FString& InString);

  /**
   * Returns a string from your system's clipboard.
   *
   * @param Return String stored in your system's clipboard.
   */
  UFUNCTION(
    BlueprintPure,
    meta = (
      DisplayName="Get System Clipboard",
      Keywords = "Paste string from clipboard"
    ),
    Category = "FileSDK | Clipboard"
  )
  static void PasteClipboard(FString& Return);
};

```